hail/python/hailtop/hailctl/batch/log.py

```diff
@@ -0,0 +1,25 @@
+from .batch_cli_utils import get_job_if_exists
+
```

Comment:
FYI -- I'm in the middle of adding pod status #6489. Whomever goes in last should probably add that.